tasks1.js

```diff
@@ -0,0 +1,107 @@
+// 1
+function convertMinutesToHours(minutes) {
+  return (Math.floor(minutes / 60)) + ':' + (minutes % 60);
+}
+console.log(convertMinutesToHours(75));
+
+
+// 2
+function isTextUppercase(text) {
+  return /^[A-Z]*$/.test(text)
+}
+console.log(isTextUppercase('START'));
+
+
+// 3
+function isDivisible(n, x, y) {
+  if (n % x === 0 && n % y === 0) {
+    return true;
+  } else return false;
+}
+console.log(isDivisible(3,3,4));
+
+
+// 4
+function getBillboardPrice(word, price) {
+  const wordLength = word.trim().length;
+  return wordLength * price;
+}
+
+console.log(getBillboardPrice('Hello world!', 10));
+
+
+// 5
+function makeOppositeNumer(number) {
+  return number - (number*2);
```

Comment:
Smart one. Please be consistent in code formatting
```suggestion
  return number - (number * 2);
```
Also you can multiply by `-1`